src/App.tsx

```diff
@@ -149,8 +149,8 @@ export const App: React.FC = () => {
           <div className="nav_container">
             {!flyToId && (
               <div className="orbit_title_container">
-                {orbitIds.map((id, index) => {
-                  return <OrbitTitleGroup key={id} id={id} tabIndex={index} />;
+                {orbitIds.map((id) => {
+                  return <OrbitTitleGroup key={id} id={id} tabIndex={0} />;
```

Comment:
adding a `tabIndex` value of higher than 0 apparently just forces the browser to tab around the page out of order and should be used very sparingly.  `tabIndex: 0` means it should be accessible in the order it appears on the html page.

`tabIndex: -1` means the element should not be accessible with the tab key.